util/ci/common.sh

```diff
@@ -34,6 +34,9 @@ install_linux_deps() {
 
 # macOS build only
 install_macos_deps() {
+	# Uninstall the bundled cmake, it is outdated, and brew does not want to install the newest version with this one present since they are from different taps.
+	brew uninstall cmake
```

Comment:
(ignore failures in case this becomes no longer needed in the future)
```suggestion
	brew uninstall cmake || :
```